cmd/bosun/sched/alertRunner.go

```diff
@@ -39,15 +48,32 @@ func (s *Schedule) RunAlert(a *conf.Alert) {
 	s.checksRunning.Add(1)
 	// ensure when an alert is done it is removed from the wait group
 	defer s.checksRunning.Done()
+	var lastCheckID int64 = -1
```

Comment:
each alert tracks the last id that it ran a check on. we actually could dispense with the field and just track lastCheckTime in the same way. Good point.